rust-src/concordium_base/src/id/sigma_protocols/common.rs

```diff
@@ -294,16 +301,16 @@ pub fn prove<R: rand::Rng, D: SigmaProtocol>(
     prover: &D,
     secret: D::SecretData,
     csprng: &mut R,
-) -> Option<SigmaProof<D::ProverWitness>> {
-    let (point, state) = prover.commit_point(csprng)?;
+) -> Option<SigmaProof<D::Response>> {
+    let (point, state) = prover.compute_commit_message(csprng)?;
```

Comment:
same comment: maybe we can make the code clearer by renaming `point` to `commit_message` here (although point is also fine)